smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -42,7 +42,7 @@ task FeatureCountsUniqueMapping {
   }
   runtime {
     docker:"humancellatlas/star_dev:v1"
```

Comment:
Do we have a standard tagging method for docker images? I'm looking and I see a mix of:
1. latest
2. v.major
3. v.major.minor x2